app/javascript/packs/restaurantsViewScreen/updateableSelectBox.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default class UpdateableSelectBox extends React.Component {
+  handleChange = (e) => {
+    this.props.filterChange({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  render () {
+    var options = this.props.values.map((selectLine) => {
+      var value = Object.keys(selectLine)[0];
+      return (<option value={value} key={value}>{selectLine[value]}</option>);
+    });
+    var label = this.props.filterName.replace(/([A-Z])/g, ' $1').replace(/^./, function(str){ return str.toUpperCase(); })
```

Comment:
what's this supposed to do? feels like a very specific assumption on how the label is supposed to be presented, not very generic. why not have the client pass it as a prop?